packages/cyberstorm/src/components/componentStyles/Button.module.css

```diff
@@ -22,7 +22,7 @@
 /* Styles for Button's text */
 .label {
   font: var(--font-heading);
-  font-weight: var(--font-weight-semi-bold);
+  font-weight: var(--font-weight-600);
```

Comment:
Is this (and the other font-weight changes in other files) here on purpose? They seem unrelated to the PR. If they're here on purpose, please make sure what we want to actually use the numbers instead of text-based names (we might want to, I can't keep up with this stuff 😅 )

Reply:
Not really to do with this, but just overall a change that was nice to do. We discussed with @gkurck about using the numbers instead of xs, 3xs etc. Probably should be done separately, yeah, but this is a nice conversation starter ;D

Reply:
Not sure if your comment means that this should wait gkurck's input so I'll leave this comment unresolved for now. Mark resolved at your discretion.

Reply:
This has been written down and will be handled separately, I'll leave this in, as it will be gone over and changed back if needed.